raiden/storage/migrations/v20_to_v21.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+from raiden.storage.sqlite import SQLiteStorage
+
+SOURCE_VERSION = 20
+TARGET_VERSION = 21
+
+
+def _transform_snapshot(raw_snapshot: str) -> str:
```

Comment:
A short docstring on what this update touches on the snapshot would be nice. Same for state changes. Short -- since the code is explanatory, but stll would be nice to read a docstring.